### DISCUSSION TITLE
Preflight MCDU Setting for SPEED to be in Managed Mode

### DISCUSSION THREAD

**Discussion by muskysim** [flybywiresim/aircraft#7108] [Help] [answered]
I normally use Experimental Version but same thing happened with Development Version

Have been using this great mod since day 1 and usually when something stops working its down to a new realism feature that gets added that I do not know about and I think this is probably the case here.

My problem is that when I fill in the MCDU pages preflight the way I have always done. I would find both SPEED & HEADING to be in Managed Mode as long as FD was on. But now nothing I try will get SPEED mode to go into Managed Mode. When I press knob it briefly displays the 4 dashes then goes back to Select Mode showing 100.

The only item I believe I am not filling in the MCDU is the wind data. 

So I would appreciate it if someone can enlighten me into what exactly needs to be entered preflight into MCDU to allow SPEED to be set in  managed mode.

Thanks again for this great mod

**Comment by aguther** [comment, marked as answer]
Hello,

I just tried to reproduce this with lastest experimental version, and I'm unable to do so. I tried starting on the runway and in parking. Once the aircraft is powered, the following steps where enough to arm NAV and switch speed to managed:

- ensure at least one FD is engaged.

- enter origin and destination airport
- select runway for take-off
-> NAV arms

- switch to PERF page
- enter flaps configuration, i.e. 1
- enter V2
-> speed switches to managed

**Reply by muskysim** [reply]
Thankyou for the quick response, It must be some setting I have on a switch or a button that forcing SPEED into Select mode. I have been using FSUIPC7 to programme functions onto buttons so I know exactly where to look .

Thanks for your help

**Comment by muskysim** [comment]
Just to close this completely and for future reference. Problem was caused by my Bravo Throttle AP Mode selector rotary  being set to IAS. mode. I do not use the AP functions of the  Bravo for FBW 320  but the default setting for this was pulling the knob every time I pushed it. 

And finally a suggestion for your installer. Surely its about time you had a option to Donate button on the installer. This mod goes way beyond freeware.

Thanks

**Reply by aguther** [reply]
Thanks for the feedback!
